libs/test/SmartStoreTest/src/com/salesforce/androidsdk/store/SmartStoreTest.java

```diff
@@ -44,12 +44,13 @@
 import org.json.JSONObject;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 
 /**
- * Abstract super class for plain and encrypted smart store tests
+ * Main test suite for SmartStore
```

Comment:
All the relevant tests come now in two flavors: with regular indexes (string/floating etc) and with the new json1 index